src/components/ui/Image/Image.tsx

```diff
@@ -1,32 +1,40 @@
 import { memo } from 'react'
-import { Helmet } from 'react-helmet-async'
+import Head from 'next/head'
```

Comment:
Drop this component in favor of next/image

Reply:
we can do this in a later PR